client/me/billing-history/controller.js

```diff
@@ -19,13 +19,12 @@ const sites = sitesFactory();
 export default {
 	billingHistory( context ) {
 		const BillingHistoryComponent = require( './main' );
-		const billingData = require( 'lib/billing-history-data' );
```

Comment:
Of course, I was planning to do so in a separate PR and this is one of the outstanding tasks in #10554.